core/src/main/scala/better/files/File.scala

```diff
@@ -829,6 +830,13 @@ object File {
   def currentWorkingDirectory: File =
     File("")
 
+  def newUniqueFile(): File =
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You should not name methods after their defining object](https://www.codacy.com/app/pathikrit/better-files/file/4972468862/issues/source?bid=2860326&fileBranchId=3817070#l833)